src/hooks/useApproveAndSwap.ts

```diff
@@ -136,7 +138,9 @@ export function useApproveAndSwap(
         indexTo,
         state.fromAmount,
         minToMint,
-        Math.round(new Date().getTime() / 1000 + 60 * 10),
+        Math.round(
+          new Date().getTime() / 1000 + 60 * state.transactionDeadline,
```

Comment:
Maybe do like `state.transactionDeadline || Deadlines.10` to be defensive in case the user clicks the custom input but doesn't enter anything

Reply:
I have it set if the input value is null or under 0, it defaults to 10